examples/chip-tool/commands/common/Command.cpp

```diff
@@ -90,6 +90,13 @@ bool Command::InitArgument(size_t argIndex, const char * argValue)
         break;
     }
 
+    case ArgumentType::String: {
+        char ** value   = reinterpret_cast<char **>(const_cast<char *>(reinterpret_cast<const char *>(arg.value)));
```

Comment:
Some comment here would be useful, maybe even a TODO to fix the interface: 
  - why are we killing const and why is this ok
  - why is this a char** rather than char* (for string)

Reply:
> Some comment here would be useful, maybe even a TODO to fix the interface:
> 
> * why are we killing const and why is this ok

I can keep the `const` here. My bad, I was a bit over zealous.

> * why is this a char** rather than char* (for string)

Mostly because we want to change the address of the `char * mName` member of ModelCommand instance.
`arg.value` is lying a bit here. It is declared to be `void *` but in this case it is used to store a `char **`.